### ISSUE TITLE
A11y_MicrosoftGuides_Outline_StepOptions_Screenreader: Nvda doesn’t announce the role as menu Item.

### ISSUE THREAD

**Issue by ShwetaBaglane** [nvaccess/nvda#10054]
### Steps to reproduce:
1. Open the Microsoft Dynamics 365 Guides (Preview) desktop application.
2. Sign-In into application using correct credentials.
3. Select the instance and click on Continue.
4. Navigate to any existing guides under the recent tab.
5. Start screen reader as NVDA.
6. Right Click on it to open the popup window. 

### Actual behavior:
While NVDA is on, navigate to any step within any task using tab key. Nvda announces as 'Add a step 1 of 3' but doesn’t announce the role as menu Item.

### Expected behavior:
While NVDA is on, navigate to any step within any task using tab key. Nvda should announce the role as menu Item before announcing 'Add a step 1 of 3'.

#### NVDA version:
NVDA 2018.3.2

#### Windows version:
Win10 Enterprise 1903

#### Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

#### User Impact:
Screen reader only users will face difficulty as they will not get to know that opened popup contains Menu Item.

#### MAS Reference:
MAS1.3.1
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by DrSooom** [comment]
@ShwetaBaglane: Please try to reproduce this with [NVDA 2019.2 RC2](https://ci.appveyor.com/api/buildjobs/qqofdhd6mlh8mq5u/artifacts/output/nvda_2019.2rc2.exe) (running via the Installer should be enough) and run NVDA before starting Microsoft Dynamics 365 Guides. Testing this with Narrator wouldn't be bad too. Thanks.

**Comment by ShwetaBaglane** [comment]
Tried to repro the issue as per the comment. But issue still reproduces on NVDA 2019.2 RC2 version.
Narrator announces correctly as 'MenuItem_Name  MenuItem'

**Comment by brunasilvalopes** [comment]
This is somewhat similar to this other bug: https://github.com/nvaccess/nvda/issues/10055

The issue can be seen on the Microsoft's publicly available app XAML Controls Gallery: https://www.microsoft.com/store/productId/9MSVH128X2ZT

**Steps to repro:**
1. Install and open XAML Controls App
2. On the home page, on the left menu, navigate to the option "Menus and Toolbars"
3. On the Menus and Toolbars Page, navigate to the Item MenuFlyout
4. On the MenuFlyout page, navigate to the first button called "Sort"
5. Press Enter
6. Notice that NVDA reads "By rating 1 of 3" but doesn't say that this is a menu item, even though the UIA tree says that it's a menu item.

**Screenshot of the UIA tree:**
![image](https://user-images.githubusercontent.com/13021878/62978441-1f762c00-bdd6-11e9-9403-657a255b3937.png)

**Transcript of the screenshot:**
"XAML Controls Gallery" window
"PopupHost" window​
"Popup" window​
"" menu​
"By rating" menu item​
"By match" menu item
"By distance" menu item

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
